lib/api/index.js

```diff
@@ -1,4 +1,18 @@
+'use strict'
+
+/**
+ * Handles a request to a not implemented `/api/` endpoint.
+ * @param req
+ * @param res
+ * @param next
+ */
+function notImplemented (req, res, next) {
+  res.status(404).send('API not implemented')
+}
+
 module.exports = {
-  accounts: require('./accounts'),
-  messages: require('./messages')
+  authn: require('./authn'),
```

Comment:
I hear you.
Part of the reason it's a common pattern is to avoid having to explicitly `require()` every handler in the APIs folder, since you can just require the API object, and then use `API.*` in your route setups.
I'm not attached to it, though - we can take out the various existing occurrences of it.